sdk/cognitiveservices/Azure.OpenAI.Inference/src/Models/CompletionsRequest.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using Azure.Core;
+
+namespace Azure.OpenAI.Inference.Models
+{
+    public partial class CompletionsRequest
+    {
+        public CompletionsRequest(IList<string> prompt)
```

Comment:
This seems to violate the guideline for not having settable collection members.  I'd advise discussing with the architects.

Reply:
looks like this is a customized code instead of generated code.
Is this a workaround of something?